contents/blog/drawing-hedgehogs.md

```diff
@@ -0,0 +1,64 @@
+
+When creating PostHog's mascot I took inspiration from companies such as Sentry and Hasura, both of which balance a very technical product with colourful and friendly mascots. They strike a perfect balance between playful and modern styles, which allows the user to feel at ease.
+
+Both Sentry and Hasura also use their logos as inspiration for their mascots. From here it was obvious to me which direction to go in and that the hedgehog must be brought back. 
+
+## Origins of a hedgehog
+
+In the early days of PostHog we often debated whether the hedgehog should be a main character for our brand or if it would be too distracting. Since then we've jumped from from the original 'hairy thumb' logo designed by our Founder and CEO, to focusing on 80’s pixelart, to space and rockets and right back to hedgehogs. 
+
+It’s been a process, to say the least, but an informative one. We never should have strayed from our first and one true love, the hedgehog. 
+
+**[thumb hog to present day hog]**
+
+The hard part, of course, is actually designing the hedgehog. This is no easy feat, but the secret is the same as most things at PostHog: focus on iteration and experimentation. 
+
+My process is as follows:
+
+– Don’t think too much, just draw
+– Draw a lot and be willing to make mistakes
+– Try things you think you won’t like (even if its awful at least you’ve ruled it out)
+– Don’t copy from others. You may think this will allow you to improve, but people will see through it. Originality is what makes you great.
+
+## Use a tablet
+
+Personally, I use a tablet to aid me in the drawing process. Maybe I'm biased or maybe I’m wise but there is nothing more satisfying than being able to press undo instead of starting again on paper. Though I have been known to try to pinch zoom on real-life paper. 
+
+Using a tablet also enables me to share unfinished ideas quickly, so I can expand on them later. I use ProCreate or Fresco, both of which allow for realistic drawing with a wide range of pencils, pens and even paint. 
+
+What really sets digital drawing apart from traditional design practice is that it comes with _layers_. Layers change lives. Long gone are the days of linear painting - now I can select a layer and work in isolation on that, or cut out a small section and reuse it later (because I love to recycle). Its sped up my work while giving me so much freedom to work with my creations. Would 10/10 recommend to a friend.
+
+**[insert gif of loads of hedgehog drawings]**
+
+As you can see the hedgehog went through about 18 different variations before I knew which one suited the brand the best, from large, small, angry, 3D and even square. But you need to try a lot to find what works, and even after finding that golden hog you still need to consistently iterate on what you have to strengthen that brand identity. 
```

Comment:
It would be quite interesting to get some thoughts on each of these - why you dismissed them @lottiecoxon